docs/UserGuide.md

```diff
@@ -127,195 +125,256 @@ Valid cases:
-        *  Duration / end date of foster, whichever the foster manager prefers
-        *  Identifiable physical traits of the animal
-      
-<div markdown="span" class="alert alert-primary">
-  :bulb: <b>Tip:</b> <code>NAME</code>, <code>NUMBER</code>, <code>EMAIL</code>, and <code>ADDRESS</code> are compulsory. 
-Empty compulsory values disable the save command that saves the edited fosterer data.
+*  `find` or `list` followed by `edit 3 n/John` 
+   * edits the name of the 3rd fosterer in the address book to John.
+*  `find` or `list` followed by `edit 1 p/12345678 animal/Bob` 
+  * edits the phone number and the pet name of the 1st fosterer in the address book to 12345678 and Bob respectively.
+*  `find` or `list` followed by `edit 2` 
+  * changes the view to the profile page of the 2nd fosterer in the address book since parameters are not provided.
+
+<div markdown="span" class="alert alert-warning">:exclamation: <b>Caution:</b><br/>
+Edit may cause information loss. 
+Before you type the save command, make sure there is no accidental overwrite with faulty information. 
 </div>
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Caution:** Edit may cause information loss. 
-Before you type the save command, make sure there is no accidental overwrite with faulty information. 
+
+### Editing a fosterer's detail through the profile page: `edit`
+
+Edits the details of a fosterer stored in the address book with updated information provided through the profile page.
+
+Format: `edit INDEX`
+
+Parameters:
+
+| Parameter | About                                                                                                                                                                     | Example |
+|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `INDEX`     | - The index of a fosterer displayed in the list obtained from a `list`/`find` command <br/> - At least one index must be provided<br/> - Index must be a positive integer | `1`, `2`, `3` |
+
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about the command format:**<br>
+
+*  The UI generates a profile page where details of the fosterer are shown.
+*  Aside from the details added by the add command, here is the list of fields the profile page shows:
+* `NOTE` : Notes for the user, examples include but are not limited to (optional and non-exhaustive):
+  * Health condition of the animal
+  * Duration / end date of foster, whichever the foster manager prefers
+  * Identifiable physical traits of the animal
+*  The respective updated field descriptions can be keyed in and saved by the user through the `save` command.
+  * Please refer to the section 'Viewing a fosterer's detail: Editing a fosterer’s details in profile page' for the specific steps to do so.
```

Comment:
You mean edit using profile view here right! cant find the section 'Viewing a fosterer's detail: Editing a fosterer’s details in profile page'

Reply:
Its correct...? so confused HAHAHA its the subsection under view feature which brandon created